diesel_codegen/tests/associations.rs

```diff
@@ -1,11 +1,6 @@
 #![allow(dead_code, unused_must_use)]
 
-#[cfg(feature = "sqlite")]
 type Backend = ::diesel::sqlite::Sqlite;
-#[cfg(feature = "mysql")]
```

Comment:
I'd still prefer to test this against all backends. Even if we're not generating anything that directly hits the database, we should still be sure that we're generating code that is valid for all backends